spacy/attrs.pxd

```diff
@@ -18,6 +18,7 @@ cdef enum attr_id_t:
     IS_QUOTE
     IS_LEFT_PUNCT
     IS_RIGHT_PUNCT
+    IS_CURRENCY
 
     FLAG18 = 18
```

Comment:
In order to not confuse the order of flags, we should remove this line and replace the next one with:

```python
FLAG19 = 19
```